### DISCUSSION TITLE
https://1kevinson.com/why-you-should-simply-use-java-for-loop-instead-of-stream-foreach-method/?ref=1kevinletter-newsletter

### DISCUSSION THREAD

**Discussion by giscus** [1kevinson/BLOG-TUTORIALS#21] [Announcements]
# https://1kevinson.com/why-you-should-simply-use-java-for-loop-instead-of-stream-foreach-method/?ref=1kevinletter-newsletter

In this article, I would like to discuss the differences between using the streaming API and for loops from the standpoint of long-term maintainability of the code.

https://1kevinson.com/why-you-should-simply-use-java-for-loop-instead-of-stream-foreach-method/?ref=1kevinletter-newsletter

**Comment by defofotg** [comment]
Thanks for the insight mate 👌

**Reply by 1kevinson** [reply]
It will be a pleasure @defofotg 🙂
